src/_content/posts/djangocon-us-grants.md

```diff
@@ -0,0 +1,63 @@
+---
+author: Communications Team
+category: General
+published_datetime: 2025-04-29 11:00:00
+title: "DjangoCon US Grants"
```

Comment:
i think this would be better titled as something like an article or blog post- right now it reads like a standalone page

Reply:
Any suggestions?